pages/get_involved.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
```

Comment:
This shouldn't appear here - make sure you checkout master, pull from upstream, and checkout a new branch from master.

Reply:
You fixed it, fantastic! :fireworks: